src/snapred/backend/service/LiteDataService.py

```diff
@@ -12,6 +15,8 @@ class LiteDataService(Service):
     def __init__(self):
         super().__init__()
         self.registerPath("createLiteData", self.reduceLiteData)
+        self.dataExportService = DataExportService()
+        self.dataService = LocalDataService()
```

Comment:
This service should not be able to access the `LocalDataService`.  It should instead have
``` python
self.dataFactoryService = DataFactoryService()
```

Reply:
getIPTS() does not exist in DataFactoryService. Should I add a function in DataFactoryService that calls getIPTS()?

Reply:
Add it to DataExportService instead, then

Reply:
Actually, don't add a `getIPTS` method, instead add a method that will create the path you create below